File: main/Iona/WisejPWAWebApplicationExample/InstallAppPanel.cs
using System;
using System.Drawing;

namespace WisejPWAWebApplicationExample
{
    /// <summary>
    /// Displays a custom panel at the bottom of the parent with an "Install" button that
    /// triggers the PWA install prompt on the browser.
    /// </summary>
    public partial class InstallAppPanel : Wisej.Web.UserControl
    {
        public InstallAppPanel()
        {
            InitializeComponent();
        }

        private void InstallAppPanel_Load(object sender, EventArgs e)
        {
            var parentSize = this.Parent.Size;
            this.Location = new Point(
                (parentSize.Width - Width) / 2,
                (parentSize.Height - Height - 5));

            BringToFront();
        }

        private void buttonClose_Click(object sender, EventArgs e)
        {
            Dispose();
        }

        private void buttonInstall_Click(object sender, EventArgs e)
        {
            Dispose();
        }
    }
}

File: main/Iona/WisejModelPWAProject/InstallAppPanel.vb
Imports System.Drawing

''' <summary>
'''  Displays a custom panel at the bottom Of the parent With an "Install" button that
'''  triggers the PWA install prompt On the browser.
''' </summary>
Public Class InstallAppPanel

    Private Sub InstallAppPanel_Load(sender As Object, e As EventArgs) Handles MyBase.Load

        Dim parentSize = Me.Parent.Size
        Me.Location = New Point(
            (parentSize.Width - Me.Width) / 2,
            (parentSize.Height - Me.Height - 5))

        BringToFront()

    End Sub

    Private Sub buttonClose_Click(sender As Object, e As EventArgs) Handles buttonClose.Click

        Dispose()

    End Sub

    Private Sub buttonInstall_Click(sender As Object, e As EventArgs) Handles buttonInstall.Click

        Dispose()

    End Sub

End Class
